tce/tools/scripts/install_llvm_9.sh

```diff
@@ -40,33 +40,19 @@ function fetch_llvm {
 
     if ! test -d $llvm_co_dir;
     then
-        svn  $REV_TO_FETCH co http://llvm.org/svn/llvm-project/llvm/branches/release_90 $llvm_co_dir \
-            || eexit "SVN co $REV_TO_FETCH from LLVM failed"
-#        svn -q $REV_TO_FETCH co http://llvm.org/svn/llvm-project/llvm/trunk $llvm_co_dir \
-#            || eexit "SVN co $REV_TO_FETCH from LLVM failed"
+	git clone --branch release/9.x https://github.com/llvm/llvm-project.git $llvm_co_dir\
+	    || eexit "Git clone $REV_TOFETCH from llvm failed"
     else
-#        svn up $REV_TO_FETCH $llvm_co_dir \
-#            || eexit "SVN update $REV_TO_FETCH of LLVM failed."
-        # Wipe out possible previously applied patches.
-        svn revert -R $llvm_co_dir
+	# Discard all differences with release/9.x branch
+	cd $llvm_co_dir;
+        git checkout release/9.x ||
```

Comment:
Perhaps do a git pull also to fetch any fixes to the release branch?

Reply:
agreed. that should be done. I put it in.